src/ocean/util/app/DaemonApp.d

```diff
@@ -380,6 +375,12 @@ public abstract class DaemonApp : Application,
         this.config_ext.registerExtension(this.unix_socket_ext);
         this.registerExtension(this.unix_socket_ext);
 
+        // Create and register repoenable files extension
+        this.reopenable_files_ext = new ReopenableFilesExt(this.signal_ext,
+            this.unix_socket_ext,
+            settings.reopen_signal);
+        this.registerExtension(this.reopenable_files_ext);
+
```

Comment:
Why is this now registered twice?

Reply:
Probably accident - we need to register it only once. Will fix.

Reply:
Wait, where do you see it's registered twice?

Reply:
I've just moved it to be registered after UnixSocketExt, since we need it for registering the handler.

Reply:
I guess I was colorblind and didn't see the red above... All good.